### PR TITLE
trying to pin and also fix linux build issue

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,24 +15,26 @@ build:
...
+    - libgfortran
     - toolchain  # [osx]
   run:
-    - hdf5 1.8.18|1.8.18.*
+    - hdf5 1.10.1
     - libnetcdf 4.4.*
     - jasper
     - g2clib
     - libpng >=1.6.28,<1.7
     - libtiff 4.0.*
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Can you update the `libtiff` and `libpng` to `libtiff >=4.0.3,<4.0.8` and `libpng >=1.6.22,<1.6.31` respectively?
You also need to bump the build number to get a new package.

**Comment by doutriaux1** [comment]
@ocefpaf any idea why it can't find gfortran?

I have seen this somewhere else where I actually need to symlink gfortran see:
https://github.com/UV-CDAT/conda-recipes/blob/master/cdtime/build.sh#L7-#L10
Should add this here?

**Comment by ocefpaf** [comment]
> I have seen this somewhere else where I actually need to symlink gfortran see:
https://github.com/UV-CDAT/conda-recipes/blob/master/cdtime/build.sh#L7-#L10
Should add this here?

The issue was on Windows or Linux? I'd rather fix that in the compiler package or maybe patching the build rather than hardcoding a symlink there. This is brittle and conda has some known issues with symlinks.

I'll take a closer look at this issue tomorrow.

PS: you need to bump the build number here to get a new package.

**Comment by isuruf** [comment]
`libgfortran.so` symlink is there in `gcc` package which comes with `gfortran`, but `libgfortran` doesn't have that symlink. That's because `libgfortran` is supposed to be used by packages that have been compiled with `gfortran` and therefore links to `libgfortran.so.3` instead of `libgfortran.so`
This package is trying to link to `libgfortran.so` without the need for `gfortran` which is not necessary. Best option is to add `gcc` as a build requirement.

**Comment by doutriaux1** [comment]
@ocefpaf ok I can try that as well. Thx.

**Comment by dnadeau4** [comment]
This has been fix with PR v3.0.1
https://github.com/conda-forge/libcdms-feedstock/pull/11

**Pull request closed without merging**